packages/grafana-ui/src/utils/standardEditors.tsx

```diff
@@ -63,7 +63,7 @@ export const getStandardFieldConfigs = () => {
       placeholder: 'none',
     },
 
-    shouldApply: field => field.type === FieldType.number,
+    shouldApply: () => true,
```

Comment:
is `shouldApply` called when an override is added?

maybe we should rename it to `shouldDefaultApply` and only call this on the defaults

Reply:
ah, that is probably a better approach. Will maybe be a problem for new broad override matchers, but sounds like a good idea